templates/sso-cd-image-stream.json

```diff
@@ -79,6 +79,24 @@
                             "kind": "DockerImage",
                             "name": "registry.redhat.io/redhat-sso-7-tech-preview/sso-cd-openshift:5.0"
                         }
+                    },
+                    {
+                        "name": "6",
+                        "annotations": {
+                            "description": "Red Hat Single Sign-On Continuous Delivery 6 Tech Preview image",
+                            "iconClass": "icon-sso",
+                            "tags": "sso,keycloak,redhat,hidden",
+                            "supports": "sso:6",
+                            "version": "6",
+                            "openshift.io/display-name": "Red Hat Single Sign-On 6 Continuous Delivery (Tech Preview)"
+                        },
+                        "referencePolicy": {
+                            "type": "Local"
+                        },
+                        "from": {
+                            "kind": "DockerImage",
+                            "name": "registry.redhat.io/redhat-sso-7-tech-preview/sso-cd-openshift:6"
+                        }
                     }
```

Comment:
You need to add same entry for `6.0` `name` here (basically identical with the above one, but having 6.0 in the name).  Without it, only `6` entry would be created in the RH CC.

Reply:
Yes, this is my goal. I wanted a new entry, only with `6` in its name. 

Based on your comment, I guess I did that correctly, didn't I?

Reply:
@slaskawi But per the new naming guide doc, there should be three tags: `latest`, `6`, and `6.0` in the RH CC. The current form will create just `6` one (`latest` already is there).